packages/fluent/scss/drawer/_layout.scss

```diff
@@ -158,7 +158,7 @@
             inset-block-start: 0;
             z-index: 2;
             opacity: 0;
-            transition: opacity .3s;
+            @include kendo-motion(medium-2, ease, opacity);
```

Comment:
In the [motion easings list ](https://github.com/telerik/kendo-themes/pull/4779/commits/6af3b3b826d5c098967460b3d954abed183e5946#diff-2fbd99b1fa8fb0dd6af4da97661b0d7b5c4270893c91fa6ec813c8488d507e20R23)is missing ease - please change it with the correct one